test/unit/spec/media/track/localmediatrack.js

```diff
@@ -222,6 +209,103 @@ const { defer } = require('../../../../../lib/util');
       });
     });
 
+    describe('#restart', () => {
+      combinationContext([
+        [
+          [true, false],
+          x => `when called on a ${description} that was ${x ? '' : 'not '}created using either createLocalTracks or create${description}`
+        ],
+        [
+          [undefined, { foo: 'bar' }],
```

Comment:
we should add a case for `{}` since its bit special.

Reply:
@makarandp0 I'm not sure about doing it in a unit test, since we cannot verify the behavior. We can write an integration test, but I don't think it will work if we don't run it in an environment where there is more than one device available.